### ISSUE TITLE
Tag search doesn't work on demo website

### ISSUE THREAD

**Issue by jmezach** [Augurk/Augurk#70]
It seems that entering a tag in the filter box in the top left corner on the demo website doesn't currently do anything. Not sure what's going on there, but we should probably investigate why that doesn't work anymore.

**Comment by timvandenhof** [comment]
Seems this feature was never implemented. See the not implemented note in the source of index.html at line 54:
https://github.com/Augurk/Augurk/blob/ca6a5be939831411687df9ca42df4beb3df8e65b/src/Augurk/index.html
Browsing through the history of that file, shows that the note was there since initial commit.

Maybe the resolution of this issue can be combined with issue #6 .

**Comment by jmezach** [comment]
Actually I think that this is implemented [here](https://github.com/Augurk/Augurk/blob/1e49233ff564b194733352174e5e7f76183fd5d5/src/Augurk/templates/menu.html#L32). The section that's in comments in the index.html was actually intended for #6 but that's still something we need to implement.

Not sure exactly why the tag filtering isn't working. I'm thinking this might have to do with the recent change of using libman to manage our client-side dependencies, instead of relying on NuGet packages. Maybe there's a library missing now. Will have to investigate.

**Comment by jmezach** [comment]
I just ran into this issue in the .NET Core port as well. It looks like [this line of code](https://github.com/Augurk/Augurk/blob/1e49233ff564b194733352174e5e7f76183fd5d5/src/Augurk.Api/Indeces/Features_ByTagAndBranch.cs#L34) is the problem here. Apparently RavenDB is unable to run this index with the following error:

```
failed to execute mapping function on 0.0.0/fb9b4e59-6d4e-be83-c02b-63cb32e93235. exception:
microsoft.csharp.runtimebinder.runtimebinderexception: 'object' does not contain a definition for 'take' 
at callsite.target(closure , callsite , object , int32 )
at system.dynamic.updatedelegates.updateandexecute2[t0,t1,tret](callsite site, t0 arg0, t1 arg1)
at raven.server.documents.indexes.static.generated.index_features_byproductandbranch.map_0(ienumerable`1 docs)+movenext()
at raven.server.documents.indexes.static.timecountingenumerable.enumerator.movenext() in c:\builds\ravendb-stable-4.1\src\raven.server\documents\indexes\static\timecountingenumerable.cs:line 41
at raven.server.documents.indexes.mapindexbase`2.handlemap(lazystringvalue lowerid, lazystringvalue id, ienumerable mapresults, indexwriteoperation writer, transactionoperationcontext indexcontext, indexingstatsscope stats) in c:\builds\ravendb-stable-4.1\src\raven.server\documents\indexes\mapindexbase.cs:line 65
at raven.server.documents.indexes.workers.mapdocuments.execute(documentsoperationcontext databasecontext, transactionoperationcontext indexcontext, lazy`1 writeoperation, indexingstatsscope stats, cancellationtoken token) in c:\builds\ravendb-stable-4.1\src\raven.server\documents\indexes\workers\mapdocuments.cs:line 108
```

Removing the **Take(50)** seems to resolve the issue, but I'm sure there was a reason we put that there in the first place. I just can't remember why. @marktaling Do you?

**Comment by marktaling** [comment]
The previous version of Raven we were using required to the Take so the index wouldn't be running forever. As it seems, the newer version of Raven we are using right now no longer support this.

We should be fine by just removing it, all it did was limit the number of tags to 50.

**Comment by jmezach** [comment]
This was fixed as part of #74 in the 2.8.0 release.
